src/server/processLauncher.ts

```diff
@@ -62,29 +63,31 @@ if (maxListeners !== 0)
 
 export async function launchProcess(options: LaunchProcessOptions): Promise<LaunchResult> {
```

Comment:
Instead of all the changes in this function, why not pass dummy log, empty args and tempDirectories, etc?

Reply:
Was also thinking about that but then doing it the other way around.